src/modules/api/pipeline-task/pipelines/utils.py

```diff
@@ -36,131 +26,89 @@ def load_json_from_request(request):
     raise APIBadRequestError('Failed to parse request body as valid JSON') from ex
 
 
-def log_ssl_backoff(details):
-  logger.warn(f'[TASK {details["args"][0].get("id", "no-id")}] Encountered SSLEOFError trying to start job. Trying again in {details["wait"]:00.1f}s...')
-
-def log_ssl_giveup(details):
-  logger.warn(f'[TASK {details["args"][0].get("id", "no-id")}] Encountered SSLEOFError trying to start job. Giving up. Total time elapsed: {details["elapsed"]:00.1f}s.')
-
-
 
 def get_task_handler(queue_name, *args, **kwargs):
-  mapping = {
-    DB_OPERATIONS_TASK_QUEUE_NAME: DatabaseOperationTaskHandler,
-    INDEL_PRIMER_TASK_QUEUE_NAME:  IndelFinderTaskHandler,
-    HERITABILITY_TASK_QUEUE_NAME:  HeritabilityTaskHandler,
-    NEMASCAN_TASK_QUEUE_NAME:      NemascanTaskHandler,
-  }
-  cls = mapping.get(queue_name, None)
-
-  if cls is None:
+  '''
+    Args:
+      - queue_name: The queue to run the job in.
+      - kwargs: The job payload.
+  '''
+  try:
+    cls = get_pipeline_class(queue = queue_name)
+
+  except ValueError:
     raise APIBadRequestError(f'Invalid task route {queue_name}')
 
   try:
-    return cls(*args, **kwargs)
+    return cls.lookup(kwargs['id'])
 
   except NotFoundError as ex:
     if ex.kind == Species.kind:
-      raise APIBadRequestError(f'{ cls._Entity_Class.kind } task has invalid species value') from ex
+      raise APIBadRequestError(f'{ cls.get_kind() } task has invalid species value') from ex
     else:
-      raise APINotFoundError(f'Could not find { cls._Entity_Class.kind } object wih this ID') from ex
+      raise APINotFoundError(f'Could not find { cls.get_kind() } object wih this ID') from ex
 
 
 
-@backoff.on_exception(
-    backoff.constant, SSLEOFError, max_tries=3, interval=20, jitter=None, on_backoff=log_ssl_backoff, on_giveup=log_ssl_giveup,
-)
-def start_job(payload, task_route, run_if_exists=False):
```

Comment:
Converted into the implementation for the `run` abstract method in `GCPCloudRunRunner`